src/main/java/org/broadinstitute/hellbender/tools/variantdb/nextgen/ExtractCohortEngine.java

```diff
@@ -493,6 +561,10 @@ private VariantContext createVariantContextFromSampleRecord(final GenericRecord
         alleles.add(ref);
         List<Allele> altAlleles = Arrays.stream(sampleRecord.get(SchemaUtils.ALT_ALLELE_FIELD_NAME).toString().split(SchemaUtils.MULTIVALUE_FIELD_DELIMITER))
                 .map(altAllele -> Allele.create(altAllele, false)).collect(Collectors.toList());
+
+        // NOTE: gnarly needs this it seems?
+        altAlleles.add(Allele.NON_REF_ALLELE);
```

Comment:
this is interesting. I just checked the original evoquer code and it doesn't seem to add the non-ref allele in. it might be worth understanding what the difference is.

Reply:
I think it's because we're now removing that allele during the ingest process so it's not in BQ.  We weren't doing that before